packages/components/src/Notification/ToastNotification/_docs/ToastNotification.stories.tsx

```diff
@@ -175,26 +175,27 @@ export const NoDuplicatesWithSameId: Story = {
   render: () => {
     const { addToastNotification } = useToastNotification()
 
-    useEffect(() => {
```

Comment:
[this story was failing](https://github.com/cultureamp/kaizen-design-system/actions/runs/12459144607/job/34775641714) before I made this change due to the toast was not visible.

Not sure why that was the case but I think using useEffect here might have cause some issue as I suspect [addToastNotification is not being wrapped inside a `useCallBack`](https://github.com/cultureamp/kaizen-design-system/blob/2fcf1ccee78c0808630e19bafbd8330897277843/packages/components/src/Notification/ToastNotification/context/ToastNotificationContext.tsx#L44) so having it inside `useEffect` dependency will cause it to be called many times

Reply:
creating a button and using `userEvent.click` is a much more efficient way to do it though